File: problems/26. Remove Duplicates from Sorted Array/main.go
func removeDuplicates(nums []int) int {
	k := 1
	for cur := 1; cur < len(nums); cur++ {
		if nums[cur] != nums[cur-1] {
			nums[k] = nums[cur]
			k += 1
		}
	}
	return k
}

File: problems/26. Remove Duplicates from Sorted Array/main.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k = 1
        n = len(nums)
        for i in range(1, n):
            if nums[i] != nums[i-1]:
                nums[k] = nums[i]
                k += 1
        return k
